### PR TITLE
Insert Request Implemented

### PR DESCRIPTION
**Pull request by kartikgupta0909**

### CODE REVIEW

**Comment by gauravmittal1995** [comment]
@kartikgupta0909 Whats the difference between this and https://github.com/nisargjhaveri/saathi/pull/35??

**Comment by imjalpreet** [comment]
It is working. Though it will be fully complete once we have our autocomplete part ready as now its difficult for the user to get the org id and asset id. 
@nisargjhaveri or @gauravmittal1995 You also have a look once.
@kartikgupta0909 I would request you to rebase your commits, i.e, Combine both of your commits.
If you need help you can follow this link: http://gitready.com/advanced/2009/02/10/squashing-commits-with-rebase.html

**Comment by gauravmittal1995** [comment]
@imjalpreet M not sure if we should merge these before the autocomplete widget is ready. Thats really important for these PRs. Lets review these and leave it open. As soon as that is merged, we can add it in these and then merge it.

File: app/controllers/requests.php [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+    function __construct() {
+        $this->load_model('requests_model');
+    }
+
+    function index() {
+        $this->load_view('requests/index');
+    }
+
+    function requests() {
```
Comment (nisargjhaveri) [inline app/controllers/requests.php]: Change name to `add` or something? So that the URL becomes more sensible.

**Comment by nisargjhaveri** [comment]
Looks fine, apart from autocomplete.

Merging for now, will add autocopmplete later.

**Pull request merged**